fdbserver/storageserver.actor.cpp

```diff
@@ -208,34 +208,43 @@ struct StorageServerDisk {
 
 	// SOMEDAY: Put readNextKeyInclusive in IKeyValueStore
 	Future<Key> readNextKeyInclusive(KeyRef key, IKeyValueStore::ReadType type = IKeyValueStore::ReadType::NORMAL) {
+		++(*kvScans);
```

Comment:
Can you add comment to the function for future reader?

Reply:
Done.